processing file asmFiles/jump.asm (1 of 1)
processing file asmFiles/branch.asm (1 of 14)
processing file asmFiles/count_days.asm (2 of 14)
processing file asmFiles/fib.asm (1 of 1)
--- memsim.hex	2019-09-22 18:25:10.625525214 -0400
+++ memcpu.hex	2019-09-22 18:25:10.575523528 -0400
@@ -12,27 +12,6 @@
 :04000B0015E0FFF904
 :04000C00FFFFFFFFF4
 :0400210000000001DA
-:0400220000000001D9
-:0400230000000002D7
-:0400240000000003D5
-:0400250000000005D2
-:0400260000000008CE
-:040027000000000DC8
-:0400280000000015BF
-:0400290000000022B1
-:04002A00000000379B
-:04002B000000005978
-:04002C000000009040
-:04002D00000000E9E6
-:04002E000000017954
-:04002F000000026269
-:04003000000003DBEE
-:040031000000063D88
-:0400320000000A18A8
-:040033000000105564
-:0400340000001A6D41
-:0400350000002AC2DB
-:040036000000452F52
-:0400370000006FF165
+:04002200XXXXXXXXDA
 :0403C0000000001623
 :00000001FF
processing file asmFiles/test.loadstore.asm (1 of 1)
--- memsim.hex	2019-09-22 18:25:28.794138464 -0400
+++ memcpu.hex	2019-09-22 18:25:28.711135664 -0400
@@ -10,10 +10,10 @@
 :04000900AC450008FA
 :04000A00AC47000CF3
 :04000B00FFFFFFFFF5
-:040020000000733732
-:0400210000002701B3
-:040022000000133790
-:04002300DEADBEEFA1
+:04002000XXXXXXXXDC
+:04002100XXXXXXXXDB
+:04002200XXXXXXXXDA
+:04002300XXXXXXXXD9
 :04003C000000733716
 :04003D000000270197
 :04003E000000133774
